src/test/java/org/gridsuite/modification/server/modifications/tabularmodifications/TabularVoltageLevelModificationsTest.java

```diff
@@ -0,0 +1,88 @@
+import com.powsybl.iidm.network.Network;
+import lombok.SneakyThrows;
+import org.gridsuite.modification.server.dto.*;
+import org.gridsuite.modification.server.modifications.AbstractNetworkModificationTest;
+import org.gridsuite.modification.server.utils.NetworkCreation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author AJELLAL Ali <ali.ajellal@rte-france.com>
+ */
+@Tag("IntegrationTest")
+public class TabularVoltageLevelModificationsTest extends AbstractNetworkModificationTest {
+    @Override
+    protected Network createNetwork(UUID networkUuid) {
+        return NetworkCreation.create(networkUuid, true);
+    }
+
+    @Override
+    protected ModificationInfos buildModification() {
+        List<ModificationInfos> modifications = List.of(
+                VoltageLevelModificationInfos.builder().equipmentId("v1").nominalVoltage(new AttributeModification<>(300., OperationType.SET)).highVoltageLimit(new AttributeModification<>(400., OperationType.SET)).lowVoltageLimit(new AttributeModification<>(299., OperationType.SET)).build(),
+                VoltageLevelModificationInfos.builder().equipmentId("v2").nominalVoltage(new AttributeModification<>(300., OperationType.SET)).highVoltageLimit(new AttributeModification<>(400., OperationType.SET)).lowVoltageLimit(new AttributeModification<>(299., OperationType.SET)).build()
+        );
+        return TabularModificationInfos.builder()
+                .modificationType("VOLTAGE_LEVEL_MODIFICATION")
+                .modifications(modifications)
+                .stashed(false)
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        List<ModificationInfos> modifications = List.of(
+                VoltageLevelModificationInfos.builder().equipmentId("v1").nominalVoltage(new AttributeModification<>(500., OperationType.SET)).highVoltageLimit(new AttributeModification<>(502., OperationType.SET)).lowVoltageLimit(new AttributeModification<>(499., OperationType.SET)).build(),
+                VoltageLevelModificationInfos.builder().equipmentId("v2").nominalVoltage(new AttributeModification<>(500., OperationType.SET)).highVoltageLimit(new AttributeModification<>(502., OperationType.SET)).lowVoltageLimit(new AttributeModification<>(499., OperationType.SET)).build()
+        );
+        return TabularModificationInfos.builder()
+                .modificationType("VOLTAGE_LEVEL_MODIFICATION")
+                .modifications(modifications)
+                .stashed(false)
+                .build();
+    }
+
+    @Override
+    protected void assertAfterNetworkModificationCreation() {
+        assertEquals(300., getNetwork().getVoltageLevel("v1").getNominalV(), 0.001);
+        assertEquals(300., getNetwork().getVoltageLevel("v2").getNominalV(), 0.001);
```

Comment:
You can test also low/ligh voltage limit

Reply:
Done.